packages/react-components/react-tags/src/components/Tag/useTag.tsx

```diff
@@ -43,17 +56,20 @@ export const useTag_unstable = (props: TagProps, ref: React.Ref<HTMLElement>): T
       ref,
       ...props,
     }),
-    content: resolveShorthand(props.content, { required: true }),
-    avatar: resolveShorthand(props.avatar),
+    avatarSize: tagAvatarSizeMap[size],
+    avatarShape: tagAvatarShapeMap[shape],
+    media: resolveShorthand(props.media),
+
+    content: resolveShorthand(props.content, { required: !!props.primaryText || !!props.children }),
```

Comment:
if `primaryText` slot is always required, then shouldn't `content` be required?

Reply:
good catch, let me update this

Reply:
Done :)